engine/src/main/java/org/terasology/rendering/nui/layers/ingame/metrics/DebugOverlay.java

```diff
@@ -170,6 +162,15 @@ public Boolean get() {
         metricsLabel = find("metrics", UILabel.class);
     }
 
+    private String getPing() {
+        if (localPlayer.getClientEntity().hasComponent(PingStockComponent.class)) {
+            PingStockComponent pingStockComp = localPlayer.getClientEntity().getComponent(PingStockComponent.class);
```

Comment:
This could be further simplified by first getting pingStockComp and then checking if the result is != null. Saves one lookup and simplifies the code.